modules/ztp-checking-du-cluster-config.adoc

```diff
@@ -0,0 +1,518 @@
++
+[source,terminal]
+----
+$ oc logs $PTP_POD_NAME -n openshift-ptp -c linuxptp-daemon-container
+----
++
+.Example output
+[source,terminal]
+----
+phc2sys[56020.341]: [ptp4l.1.config] CLOCK_REALTIME phc offset  -1731092 s2 freq -1546242 delay    497
+ptp4l[56020.390]: [ptp4l.1.config] master offset         -2 s2 freq   -5863 path delay       541
+ptp4l[56020.390]: [ptp4l.0.config] master offset         -8 s2 freq  -10699 path delay       533
+----
+
+. Check that the SR-IOV configuration is correct by running the following commands:
+
+.. Check that the `disableDrain` value in the `SriovOperatorConfig` resource is set to `true`:
++
+[source,terminal]
+----
+$ oc get sriovoperatorconfig -n openshift-sriov-network-operator default -o jsonpath="{.spec.disableDrain}{'\n'}"
+----
++
+.Example output
+[source,terminal]
+----
+true
+----
+
+.. Check that the `SriovNetworkNodeState` sync status is `Succeeded`:
++
+[source,terminal]
+----
+$ oc get SriovNetworkNodeStates -n openshift-sriov-network-operator -o jsonpath="{.items[*].status.syncStatus}"
+----
++
+.Example output
+[source,terminal]
+----
+Succeeded
+----
+
+.. Verify that the expected number and configuration of virtual functions (`Vfs`) under each interface configured for SR-IOV is present and correct in the `.status.interfaces` field. For example:
++
+[source,terminal]
+----
+$ oc get SriovNetworkNodeStates -n openshift-sriov-network-operator -o yaml
+----
++
+.Example output
++
+[source,yaml]
+----
+apiVersion: v1
+items:
+- apiVersion: sriovnetwork.openshift.io/v1
+  kind: SriovNetworkNodeState
+...
+  status:
+    interfaces:
+  ...
+    - Vfs:
+      - deviceID: 154c
+        driver: vfio-pci
+        pciAddress: 0000:3b:0a.0
+        vendor: "8086"
+        vfID: 0
+      - deviceID: 154c
+        driver: vfio-pci
+        pciAddress: 0000:3b:0a.1
+        vendor: "8086"
+        vfID: 1
+      - deviceID: 154c
+        driver: vfio-pci
+        pciAddress: 0000:3b:0a.2
+        vendor: "8086"
+        vfID: 2
+      - deviceID: 154c
+        driver: vfio-pci
+        pciAddress: 0000:3b:0a.3
+        vendor: "8086"
+        vfID: 3
+      - deviceID: 154c
+        driver: vfio-pci
+        pciAddress: 0000:3b:0a.4
+        vendor: "8086"
+        vfID: 4
+      - deviceID: 154c
+        driver: vfio-pci
+        pciAddress: 0000:3b:0a.5
+        vendor: "8086"
+        vfID: 5
+      - deviceID: 154c
+        driver: vfio-pci
+        pciAddress: 0000:3b:0a.6
+        vendor: "8086"
+        vfID: 6
+      - deviceID: 154c
+        driver: vfio-pci
+        pciAddress: 0000:3b:0a.7
+        vendor: "8086"
+        vfID: 7
+----
+
+. Check that the cluster performance profile is correct. The `cpu` and `hugepages` sections will vary depending on your hardware configuration:
++
+[source,terminal]
+----
+$ oc get PerformanceProfile openshift-node-performance-profile -o yaml
+----
++
+.Example output
+[source,yaml]
+----
```

Comment:
Assume it is intended to only include the spec of the output in doc, should the same the be done for other CRs for consistency? e.g., clusterlogging CRs, tuned, ....

Reply:
I think it is good to limit the output to the relevant sections that the user is checking. If there is nothing in the "status" that the user is validating it's better to leave it out (maybe show "status: ..." to indicate it is truncated?)